stories/index.stories.js

```diff
@@ -154,6 +155,34 @@ storiesOf('Label', module)
     </div>
   )));
 
+storiesOf('Glyphicon', module)
+  .add('Primary button with icon', withInfo('')(() => (
+    <div>
+      <Button bsSize="small" bsStyle="primary">
+        <Glyphicon glyph="plus " />
+        {'  Add a Release'}
```

Comment:
It's better if we change the CSS to space the icon correctly. Requiring whitespace in the content in order to use the component properly isn't expected (further, HTML collapses consecutive spaces into a single space).